src/edu/sjsu/fwjs/Type.java

```diff
@@ -0,0 +1,91 @@
+package edu.sjsu.fwjs;
```

Comment:
The class seems good, though we should probably change the package name to edu.sjsu.stratagem.